### ISSUE TITLE
Wrong file import and no runtime added

### ISSUE THREAD

**Issue by Seeveen** [Radarr/Radarr#2844]
**Describe the bug**
So Radarr downloaded Escape Plan 2 Hades 2018 DVDRip XviD AC3-EVO, on completion Radarr imported the sample file which is 90 mb instead of importing the movie , also Radarr is not able to import the runtime from IMDB or TVDB

**Expected behavior**
On finish download Radarr import the sample instead of the movie file, this might because the sample file is too big, also Radarr will not import ruining time

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Windows 10 x64
  - Version: 0.2.0.1120 - 10 Jun 2018

Log: 

AcceptableSizeSpecification | [tt6513656][Escape Plan 2: Hades (2018)] has no runtime information using median movie runtime of 110 minutes.
-- | --

**Comment by hotio** [comment]
Have you disabled mediainfo? Radarr shouldn't import the sample because it uses 110 minutes as per you log.

**Comment by Seeveen** [comment]
if you referring media info 
![rr](https://user-images.githubusercontent.com/24566404/41464672-0658f0e0-7093-11e8-91ff-f7980b171f17.png)

from what i understand Radarr did not import the sample because the runtime was because file size
![rrr](https://user-images.githubusercontent.com/24566404/41464707-27a696e4-7093-11e8-86b4-41c986f1b33e.png)

**Comment by hotio** [comment]
I can confirm that it indeed shows 0 minutes, but I get this for quite a few.....

**Comment by Seeveen** [comment]
yes but not for already downloaded movies, and this movie now trying every 10 minutes to match the runtime, is there any fix ?

**Comment by hotio** [comment]
If themoviedb does not update the info on their api, there's not much for radarr to do, besides waiting.

But, you seem to have a different problem going on, radarr picking a sample instead of the actual movie.

**Comment by hotio** [comment]
Did a test just now, the 0 minutes runtime has no effect on the import, all worked correct, did not have a sample file in the same folder tho.

**Comment by Seeveen** [comment]
like i saw above, i think reason for that was that the sample size was 90.1 MB

and about the runtime why Radarr is not picking up first from IMDB since that is more reliable ? and i checked early on on MovieDB and runtime is updated, yesterday when i look was not

**Comment by hotio** [comment]
imdb does not have an open api, and like I said above, the api has a lag probably

**Comment by hotio** [comment]
https://www.themoviedb.org/talk/55ad6d319251411e0b009403

**Comment by hotio** [comment]
Update your movie info now, the runtime will get filled.

**Comment by Seeveen** [comment]
i did, all good regarding runtime
 however i think will be good to set the threshold for **sample** to be increased on Radarr , so the sample  that are 90 mb or 100  not be imported, i never saw movies that are that low in size anyway

**Comment by hotio** [comment]
There's no way for us to know why it picked the sample without the proper logs.

**Comment by Seeveen** [comment]
That was one time only.  on that movie only, and when i posted on discord the answer was that the sample file was to big and make to trigger to copy, on other movies the sample is 2-5 mb

**Comment by hotio** [comment]
Ah yes, in that case radarr might indeed need some finetuning.

**Comment by hotio** [comment]
`if (subject.Release.Title.ToLower().Contains("sample") && subject.Release.Size < 70.Megabytes())`
so yours was 98, above 70, so according to radarr, not a sample

**Comment by Seeveen** [comment]
yes i do understand that but i think will be better if the cutoff will be above 98 mb, since there is no movie that has 90-100 mb in size, but this is mine opinion, however is not that often when you find a sample that is that big

**Comment by Qstick** [comment]
140 is used for large file qualities, but currently in code those are only ` _largeSampleSizeQualities = new List<Quality> { Quality.HDTV1080p, Quality.WEBDL1080p, Quality.Bluray1080p };`

This likely needs to be updated with all other added qualities radarr has

**Comment by Qstick** [comment]
Ref sample detection updates from Sonarr, file size backup is removed. <https://github.com/Sonarr/Sonarr/commit/8cf028e0715e352b8b1f61cc8497fa71ef8820d8#diff-b1e8e668b1366a532308125977b7cb83>

**Comment by galli-leo** [comment]
Should be relatively easy to port over commit from Sonarr, if anyone want's to do that.

**Comment by 4C3T** [comment]
Hi, sorry to bump the thread, and feel free to refer to a newer location.
I've just had trouble with Radarr downloading (via µTorrrent) this Hindi movie, and added it as "English" by force, and it took about a day to download it (Shifty seeds/peers), and finally it stated the file was 4.6 GB in size, and after browsing to it's location I find this file which has the filename of the would-be downloaded movie title, except that it differs from the original from the Torrent file (And no, no renaming or altering of files activated!).
![Crap](https://user-images.githubusercontent.com/29127941/70460537-d8175800-1ab6-11ea-9a87-6f9391457fb9.PNG)



For some weird reason it imported it while deciding which one was the movie file, at some point read the file size, saved it, only that it imported the sample file instead and removed the download from µTorrent.
The sample file size was about 12 MB, but a refresh yielded no change, still was reporting 4,6 GB. If renamed/removed it also removed it from the listing.

I had to manually re-download it and copy over the video file for it.

The movie in question was Enthiran as well as Enthiran 2.0.
_This is no serious download, just for kicks as it looked funny, best Bollywood-over-the-top-mega-action, in case anyone is wondering..._

This is the first time this has happened to me, and I've been running Radarr for about a year now.
One thing that might fudge something ut, that I can think of, is the restrictions section where I've entered "Hindi" as "Must Not Contain".
Also strange that when adding language manually, there's not a complete language list, so I wasn't able to choose Hindi for these two movies.

![Crap2](https://user-images.githubusercontent.com/29127941/70460683-33494a80-1ab7-11ea-8ecb-3351d083987b.PNG)

Reason for the "Must Not Contain" is that I don't want Radarr to automagically download some dubbed movie in various languages
